docs(site): Add metaphor-free TL;DR  to Introduction (#608)
stryker-mutator/stryker-mutator.github.io: docs/General/introduction.mdx

sidebar_position: 1
---

> **TL; DR: Mutation testing introduces changes to your code, then runs your unit tests against the changed code. It is expected that your unit tests now fail. If they don't fail for a change in the code this indicates the code is not sufficiently covered by your tests.**

Reviewer: ```suggestion
**TL; DR: Mutation testing introduces changes to your code, then runs your unit tests against the changed code. It is expected that your unit tests will now fail. If they don't fail, it might indicate your tests do not sufficiently cover the code.**
```

This is a bit more conservative. What do you think?

Reviewer 2: Yep I agree with going more conservative.
